source/Waves.cpp

```diff
@@ -781,6 +784,7 @@ void Waves::setup(EnvCond *env, const char* folder)
 				unsigned iti = 1;
 				for (unsigned int it = 0; it < nt; it++)
 				{
+					iti = it + 1;  // Set initial iti
```

Comment:
Do you know if this is needed in https://github.com/mattEhall/MoorDyn/pull/31?

Reply:
I don't know - I'm attaching the input files I used to test Matt's current v2 branch if you'd like to give it a go. I can also attempt it when I have the chance to get your fork installed and working. [current_profile_dynamic.txt](https://github.com/mattEhall/MoorDyn/files/9765766/current_profile_dynamic.txt)
[test_dynamic_currents.txt](https://github.com/mattEhall/MoorDyn/files/9765767/test_dynamic_currents.txt)

That actually reminds me of something else I wanted to check in about - last week I followed the newer documentation for installing your fork, but the CMake build was failing on the Windows setups I tried (tried 3 separate machines last week, ran into the same problem). I haven't had time to really analyze that problem/send over a complete report. I hope to find some time this week to replicate the problem and I can share more about where I am running into trouble. I think I may just be doing something stupid in my MSYS setup, but will probably be good to update docs/troubleshooting once we know what the issue is. Anyhow, I'll file a proper report when I get the time to run through it again, just wanted to give a heads up.